client/src/components/FlightResults.js

```diff
@@ -0,0 +1,105 @@
+/** @format */
+
+import React, { useState } from "react";
+import {
+	Container,
+	Grid,
+	Typography,
+	FormControl,
+	Select,
+	MenuItem,
+	InputLabel,
+} from "@material-ui/core";
+import useStyles from "../styles/FlightResults";
+import FlightAccordion from "./FlightAccordion";
+
+const FlightResults = ({ data }) => {
+	const classes = useStyles();
+
+	const prices = {
```

Comment:
if this is just dummy data for now, and is a constant, i would pull this outside of the component (for example above line 16) so you remember to remove this and this constant never gets changed

Reply:
Above line 16 are the import statements. Are you suggesting removing the import statement for the FlightAccordion?

Reply:
i suggest moving this constant above line 16 because this is a constant that never changes and is not dependant on states or component related

Reply:
Ok, I understand now. I have moved it outside of the component. Thanks.